tcms/signals.py

```diff
@@ -207,21 +207,28 @@ def handle_comments_pre_delete(sender, **kwargs):
 
 
 def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
-    """
-    Send email updates to assignee after they've been
-    assigned a bug on bug creation.
-    """
-    if not created or instance.assignee is None:
+    from tcms.core.helpers.comments import get_comments
+
+    comments = get_comments(instance)
+
+    if instance.assignee is None or comments.last() is None:
         return
 
+    recipients = list(comments.values_list("user_email", flat=True))
```

Comment:
Instead of `list` turn into a `set` to avoid duplicate addresses.

Reply:
Still valid.

Reply:
True, but this isn't a reason to not avoid duplicates in the first place, especially given that these lines of code are being modified anyway and what's required is a simple data type change.